install.js

```diff
@@ -14,6 +14,11 @@
  * limitations under the License.
  */
 
+if (process.env['PUPPETEER_SKIP_CHROMIUM_DOWNLOAD']) {
```

Comment:
any reason not to use `process.env.PUPPETEER_SKIP_CHROMIUM_DOWNLOAD`?

Reply:
Property access with string literals is done to trick type linter and avoid the `ERROR: PUPPETEER_SKIP_CHROMIUM_DOWNLOAD is never defined on process.env object`. That's a habbit from devtools codebase.

Done.